docs/source/dev-docs.rst

```diff
@@ -0,0 +1,48 @@
+##################
+Installation Guide
+##################
+
+This Document is part of the OpenDCS Software Suite for environmental
+data acquisition and processing. The project home is:
+https://github.com/opendcs/opendcs
+
+See INTENT.md at the project home for information on licensing.
+
+.. contents. Table of Contents
+   :depth: 3
+
+
+Overview
+========
+
+This is the initial start of the documentation and thus it's still quite empty
```

Comment:
maybe worth calling out the GUI main class here? maybe some other important ones

Reply:
Not sure what you mean, there are several "main gui" classes.

Reply:
Yeah that's I guess kinda my point. Hard to know where the good starting point is for development. I was thinking just "LauncherFrame" as a start

Reply:
Oh I see, yeah. I get that. The whole document could use some expansion but I'd prefer to do that later. This particular fix has been needed for a while for CWMS and it's been stopping us from doing other releases.

Reply:
agreed